kvstore/kvstore_server.h

```diff
@@ -1,6 +1,7 @@
 #include <iostream>
 #include <grpc/grpc.h>
```

Comment:
should be in its own section separated by a newline below the C++ standard library headers. 

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
ok